java/dev/checkstyle/suppressions.xml

```diff
@@ -33,10 +33,9 @@
   <suppress checks="Header" files="AutoCloseables.java|Collections2.java" />
 
   <!-- TODO: Temporarily suppress Javadoc checks -->
-  <suppress checks="Javadoc.*" files=".*" />
+  <suppress checks="javadoc.*" files=".*" />
+
+  <!-- TODO: Temporarily suppress certain check requiring many code changes -->
+  <suppress checks="NoFinalizer|OverloadMethodsDeclarationOrder|VariableDeclarationUsageDistance" files=".*" />
```

Comment:
These are the only 2 checks being suppressed:

* `NoFinalizer` is because PlasmaClient is using a `finalize` method, that should not be changed here
* `OverloadMethodsDeclarationOrder` would require a lot of code moving around. This is mostly because the vector classes have many different setXXX, setSafeXXX, getXXX methods that aren't always together, mostly because `set` and similar `setSafe` methods are placed together. Anyway, I think most of the current ordering is fine and this could be supressed for now.
* `VariableDeclarationUsageDistance` this came up quite a few times and is usually when there are many variables declared, but then used just a little further. I don't think this is a big deal, so suppressing for now.